Paintroid/src/main/java/org/catrobat/paintroid/command/implementation/DefaultCommandManager.kt

```diff
@@ -187,7 +187,7 @@ class DefaultCommandManager(
             for (index in layerCount - 1 downTo 0) {
                 val destinationLayer = layerModel.getLayerAt(index)
                 if (!checkBoxes[index]) {
```

Comment:
```suggestion
                if (destinationLayer != null && !checkBoxes[0]) {
                destinationLayer.switchBitmaps(false)
                destinationLayer.checkBox = false
            }
```
You can remove `!!` and write something like suggested above :)
Check the if block in line 196 of this file.